src/pages/rekey-reason/rekey-reason.ts

```diff
@@ -15,19 +23,47 @@ import { merge } from 'rxjs/observable/merge';
 import { SendCurrentTest } from '../../modules/tests/tests.actions';
 import { RekeyReasonUploadModel } from './rekey-reason.model';
 import { getUploadStatus } from './rekey-reason.selector';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+import {
+  getReasonForRekey,
+  getRekeyIpadIssueSelected,
+  getRekeyIpadIssueTechFault,
+  getRekeyIpadIssueBroken,
+  getRekeyIpadIssueStolen,
+  getRekeyIpadIssueLost,
+  getRekeyTransferSelected,
+  getRekeyOtherSelected,
+  getRekeyOtherReasonUpdated,
+} from '../../modules/tests/rekey-reason/rekey-reason.selector';
+import { getTests } from '../../modules/tests/tests.reducer';
+import { getCurrentTest } from '../../modules/tests/tests.selector';
 
 interface RekeyReasonPageState {
   uploadStatus$: Observable<RekeyReasonUploadModel>;
+  ipadIssueSelected$: Observable<boolean>;
```

Comment:
Also here you could have only 3 observables: `ipadIssue$`, `transfer$` & `other$`.